hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -342,7 +345,7 @@ private MemberGroupConfig createGroupConfig(String[] addressArray) {
     }
 
     private <K, V> IMap<K, V> getMap() {
-        HazelcastInstance instance = createHazelcastInstance(getConfig());
+        HazelcastInstance instance = createHazelcastInstance(new Config().setProperty("hazelcast.logging.type", "log4j"));
```

Comment:
Sorry, wanted to state it should be like this  `HazelcastInstance instance = createHazelcastInstance(getConfig())`